packages/node_modules/@ciscospark/redux-module-conversation/src/reducer.js

```diff
@@ -11,7 +11,7 @@ import {
   UPDATE_MERCURY_STATE
 } from './actions';
 
-export const initialState = {
+export const initialState = new Map({
   activities: new OrderedMap(),
   id: null,
   lastAcknowledgedActivityId: null,
```

Comment:
Should we put all of the `isX` items into the status object to match the other reducers?